ec2/README.md

```diff
@@ -49,7 +49,21 @@ This directory contains ansible automation to create a single ec2 host and insta
 ## To create the environments
 1. `cp sample.my_vars.yml my_vars.yml`
 2. `./create_infra.sh`
-3. `./setup_node.sh`
+3. 
+### To create jboss wildfly instance
+
+* Make sure to uncomment the role `jboss-wildfly` and comment `jboss-eap` in `setup_node.yml`
+
+Run `./setup_node.sh`
+
+### To create jboss EAP instance
+
+* We will be using offline installation for this role.Please [download](https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=appplatform&version=7.4) `jboss-eap-7.4.zip` from the RH Customer Portal and place it in `../ec2/jboss-eap-7.4.zip`. Make sure the file type is `zip`.
+
+* Make sure to uncomment the role `jboss-eap` and comment `jboss-wildfly` in `setup_node.yml`
+
+Run `./setup_node.sh`
```

Comment:
Small typo: s/setup_node.yml/setup_nodes.yml

Reply:
updated. Thank you.